src/02_datatypes.py

```diff
@@ -14,8 +14,10 @@
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+x + int(y)
 
 # Write a print statement that combines x + y into the string value 57
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+str(x) + y
```

Comment:
You have this! But you have to write out the print statement. 
print(str(x) + y)